README.md

```diff
@@ -1,44 +1 @@
-# A Stylish Music Player
-
-[![license](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/ryanhoo/StylishMusicPlayer#license)
-[![platform](https://img.shields.io/badge/platform-Android-yellow.svg)](https://www.android.com)
-[![API](https://img.shields.io/badge/API-16%2B-brightgreen.svg?style=flat)](https://android-arsenal.com/api?level=16)
-[![gitter](https://img.shields.io/gitter/room/nwjs/nw.js.svg)](https://gitter.im/stylist-music-player/bug-report)
-[![PRs Welcome](https://img.shields.io/badge/prs-welcome-brightgreen.svg)](http://makeapullrequest.com)
-
-![Artboard](materials/Artboard.png)
-
-## TODO
-
-- Lyric Support
-- Settings
-
-## Acknowledgements
-
-Thanks to these projects and libraries:
-
-**Libraries**
-
-- [RxJava](https://github.com/ReactiveX/RxJava)
-- [RxAndroid](https://github.com/ReactiveX/RxAndroid)
-- [Retrofit](https://github.com/square/retrofit)
-- [Butter Knife](https://github.com/JakeWharton/butterknife)
-- [Calligraphy](https://github.com/chrisjenx/Calligraphy)
-- [LiteOrm](https://github.com/litesuits/android-lite-orm)
-
-**Design**
-
-- [Material icons](https://design.google.com/icons/)
-
-
-## License
-
-> The MIT License (MIT)
-
-> Copyright (c) 2016 Ryan Hoo
-
-> Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
-
-> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
-
-> THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+fork from stylish music player
```

Comment:
Maybe you can change the README.md to the old version and add your changes to it?